### ISSUE TITLE
Unit Test Accessibility Assertions

### ISSUE THREAD

**Issue by Allcharles** [QutEcoacoustics/workbench-client#395]
Need to create accessibility tests for common elements such as images, buttons, and links. This way bad practices are less likely to get into production, and verifying accessibility support will be improved.

>Create a github issue for this (backlog, triage medium), really should be done. Other assertions may also benefit from adding accessibility expectations.

_Originally posted by @Allcharles in https://github.com/QutEcoacoustics/workbench-client/pull/385_
